### PR TITLE
deleted message column, added action column, refactored some code whi…

### PR DESCRIPTION
**Pull request by elioncho**
…ch needs more refactoring, will do it in next todo

### CODE REVIEW

**Comment by elioncho** [comment]
[Quitar columna message y agregar action con 2 posibles valores, :hit y :miss](https://trello.com/c/JhJKfB13/17-quitar-columna-message-y-agregar-action-con-2-posibles-valores-hit-y-miss)

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -22,9 +22,9 @@ def slot_state(row, column)
   end
 
   def friendly_move_message(move)
-    msg = "fila: #{(move.row + 65).chr}, columna: #{move.column + 1}"
-    msg += " - #{move.message}" unless move.message.blank?
-    msg
+    "fila: #{(move.row + 65).chr}, columna: #{move.column + 1} - #{message.action}"
+    # msg += " - #{move.message}" unless move.message.blank?
```
Comment (EdgarOrtegaRamirez) [inline app/helpers/application_helper.rb]: Please delete commented out code.

File: app/models/move.rb [hunk 2]
```diff
@@ -1,4 +1,6 @@
 class Move < ApplicationRecord
+  MISS = :miss
```
Comment (EdgarOrtegaRamirez) [inline app/models/move.rb]: Please add prefix to the constants, `ACTION_MISS`.

File: app/views/moves/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<div class="panel-body">
```
Comment (EdgarOrtegaRamirez) [inline app/views/moves/_form.html.erb]: @elioncho This change doesn't fit in this PR.

**Pull request merged**